code/modules/ninja/suit/mask.dm

```diff
@@ -3,19 +3,18 @@
 
 Contents:
 - The Ninja Space Mask
+- Ninja Space Mask speech modification
 
 */
 
-/obj/item/clothing/mask/gas/voice/space_ninja
+
+
+
+/obj/item/clothing/mask/gas/space_ninja
 	name = "ninja mask"
 	desc = "A close-fitting mask that acts both as an air filter and a post-modern fashion statement."
-	icon_state = "s-ninja(norm)"
+	icon_state = "s-ninja"
```

Comment:
Note to self; make sure all the icons, both object and on mob are what is from TG; dunno if this has been updated or not.

Reply:
Note to self: this was done.